biz.aQute.repository/test/aQute/bnd/repository/maven/provider/WorkspaceTest.java

```diff
@@ -57,6 +64,37 @@ public void testEnv() throws Exception {
 		assertNotNull(workspace);
 		assertNotNull(repo);
 		System.out.println(workspace.getBase());
+
+		// check repo tags
+		// repos should have the 'resolve' tag by default if no tag is specified
+		List<Repository> repos = workspace.getPlugins(Repository.class);
+		List<Repository> resolveRepos = workspace.getPlugins(Repository.class, RepoTags.resolve.name());
```

Comment:
The enum usage instead of a String constant does not really help, don't you agree?